templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Car</h1>
+  
+  <p>Are you sure want to delete the car: {{ car }}?</p>
+  <p><i>All data with car also delete</i></p>
+  <form action="" method="post">
+  {% csrf_token %}
+  <input type="submit" value="Yes, delete" class="btn btn-danger">

```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.